public/assets/css/main.css

```diff
@@ -54,9 +54,6 @@ i,
 a {
   display: inline-block; }
 
-span {
-    display: block; }
-}
```

Comment:
This was a syntax error in the `main.css`. Moreover, this doesn't make sense to change the default `inline `behavior for the `<span>` tag here to `block`. Doesn't it kill the purpose of span?

Reply:
I’m so confused how this error has been here since February…? But definitely seems like there is an extra curly brace. Did you poke around the pages and make sure this removal didn’t cause any major styling issues?

Reply:
Really sorry my ignorance here, @littleforest. 

- I was at the thought that this span style was being used only for the asterisk signs for them to be on the new line
- This fact skipped my mind that it's a global style I'm changing :sweat: 
- I validated now and found many styling issues due to this. :cry: 
- Anyway, I'll add a custom CSS class to override this style for the required asterisk signs here.
- However, I believe we should have an enhancement issue where this style should be removed and all the styling issues should be handled accordingly as well.
- By nature `span` is an inline element and changing this nature to `block` globally kills the purpose of this element.
- We have other alternatives to achieve this behavior like paragraph tag, divs etc..
- What do you say?

Reply:
@chahmedejaz I agree that it is odd that span is set to display: block. Do you want to create a new ticket and see about making that change? You will probably have to touch a lot of pages/styles.

Reply:
> @chahmedejaz I agree that it is odd that span is set to display: block. Do you want to create a new ticket and see about making that change? You will probably have to touch a lot of pages/styles.

Sure, I'd love to work on this refactor in a new ticket. :)

Reply:
@littleforest - The respective ticket to handle this has been created: 
https://github.com/rubyforgood/casa/issues/5377
Please review and let me know if something is missing. Thanks.